test/Writer/Extension/ITunes/FeedTest.php

```diff
@@ -209,8 +209,8 @@ public function testSetKeywordsThrowsExceptionIfMaxKeywordsExceeded()
         }, \E_USER_DEPRECATED);
         try {
             $feed->setItunesKeywords($words);
-            $this->fail('Expected exception when setting more keywords than allowed');
         } catch (ExceptionInterface $e) {
+            $this->assertInstanceOf(ExceptionInterface::class, $e);
```

Comment:
I am not sure about this change. In case there will be no any exception the test will pass. Right?

Reply:
We need this order here to ensure the restore of the error handler.

Reply:
My point is that we need some sort of fail in case `setItunesKeywords` does not throw any exception.

And I believe finally will be executed anyway - I mean if we keep `$this->fail` call in `try`.